n2kMappings.js

```diff
@@ -12,6 +12,39 @@ function getMmsiContext(n2k) {
 }
 
 exports.mappings = {
+  //Battery Voltage
+  '127508': [{
+    source: 'Voltage',
+    node: 'electrical.batteries.0.capacity.stateOfCharge',
+    filter: function(n2k) {
+      return n2k.fields['Battery Instance'] === 0;
+    }
+  }, 
+{
+    source: 'Voltage',
+    node: 'electrical.batteries.1.capacity.stateOfCharge',
```

Comment:
https://github.com/SignalK/specification/blob/master/keyswithmetadata.json#L371-L397

BTW I would like to hear from you as a newcomer: how did your discovery of Signal K go? In your view what kind of documentation should we have?